### PR TITLE
fix(alicefate): default feedback popover closed

### PR DESCRIPTION
**Pull request by alicefate**

### CODE REVIEW

File: apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js [hunk 1]
```diff
@@ -41,10 +41,10 @@ const IdeasBetaFeedbackModal = () => {
     localStorage.setItem('dl_Ideas_isVisited', true);
   }
 
-  useEffect(() => {
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js:44]: could we just delete this altogether?

Comment (dbradham) [reply apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js:44]: @alicefate friendly nudge here : )

Comment (alicefate) [reply apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js:44]: @dbradham done

**Review by dbradham — APPROVED** [review]
Looks good!

**Pull request merged**